app/ide-desktop/lib/dashboard/src/layouts/AssetPanel.tsx

```diff
@@ -139,7 +139,7 @@ export default function AssetPanel(props: AssetPanelProps) {
               dispatchAssetEvent={dispatchAssetEvent}
             />
           )}
-          <AssetVersions hidden={tab !== AssetPanelTab.versions} item={item} />
```

Comment:
this was intentional, without it being always visible, then the versions take a while to load.

Reply:
And yet it doesn't, take a while to load + we store the current tab in localstorage, so it doesn't prevent users from seeing a loader. If we decide that we want to see versions _earlier_ , we might want to use prefetching(https://tanstack.com/query/latest/docs/framework/react/guides/prefetching#prefetch-in-components)